### ISSUE TITLE
Scenario where calculated ratio can go up to the sky

### ISSUE THREAD

**Issue by OlivierGaland** [transmission/transmission#588]
Hello,

I'm not a dev, rather an user of transmission, I noticed a case were the ratio can be huge like  > 10000.

Scenario involved is (as best I can describe it) : 
- a transmission working since monthes with torrents on a tracker1
- several locations used to store torrents (download_1, download_2 ... )
- tracker1 change his domain name and ask users to alter all their torrent file to put tracker2 inside

They are several methods to do this request, one was (and reproduced this issue) is to uninstall/reinstall completly transmission (losing it current stats). Then putting back all the torrent in queue directory

Once the newly reinstalled transmission is launched (and all the torrents put again in queue directory) : 
 - there will be a short period of time where all the torrents target location will be in default download dir   
 - some download could start altering some stats (I guess it will add some low amount to downloaded bytes) because the download dir is the same (default)  for all torrents added in queue dir
 - during this time the user will pause all torrents and put all the torrents in their correct directory (download_1, download2 ...) using remoteGUI 
 - once this is done the user will start all paused torrents (after doing a verify)

After all this procedure is over  we can notice for the few torrents that have started to download before the user pause them all :
  - the ratio is going to extremly high values when peers are starting to get pieces from user the value could reach thousands

The ratio keeps "normal" values for other torrents (besides the fact we have lost all previous stats due to reinstall)

Personnal point of view :
 My understanding of this behavior is the ratio is computed with upload/download bytes stats, in case download = 0 the system use the file size as default to avoid a 0 division
In the scenario described the download is not 0 (and very low) and is probably used to compute the ratio, thus giving huge numbers. 

Solution proposal : 
 I guess in case a verify is done on a torrent putting the file size in downloaded bytes stat could avoid this issue. (I can't assess the side-effects if this is done)

Of course this is not really a bug, rather an issue caused by user that did the maintenance. And in any case more a cosmetic change than an urgent problem. 

Best regards,
Olivier

**Comment by pqhf5kd** [comment]
I've noticed the same issue when moving data from one machine to another.
<img width="305" alt="image" src="https://user-images.githubusercontent.com/20027419/63152499-43bb4e00-c00c-11e9-8790-aeeec15e5ff5.png">

**Comment by pqhf5kd** [comment]
Also mentioned here #276

**Comment by ckerr** [comment]
Duplicate of https://github.com/transmission/transmission/issues/276
